definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -54,5 +54,12 @@
     "CaseEventID": "tempEditDerivedAddresses",
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "29/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "amendPetitionForRefusalRejection",
```

Comment:
I think the CaseEventID should be: amendPetitionForRefusalSol

Reply:
ouch! copied over the wrong one. Thanks

Reply:
Corretcted